ament_index_python/ament_index_python/resources.py

```diff
@@ -49,7 +50,7 @@ def _name_is_invalid(resource_name):
     return ('/' in resource_name) or ('\\' in resource_name)
 
 
-def get_resource(resource_type, resource_name):
+def get_resource(resource_type: str, resource_name: str) -> Tuple[str, Union[str, os.PathLike[str]]]:
```

Comment:
Now that we've changed get_search_paths to return a list of strings, this can be:

```suggestion
def get_resource(resource_type: str, resource_name: str) -> Tuple[str, str]:
```